q2_moshpit/busco/busco.py

```diff
@@ -141,72 +154,97 @@ def _visualize_busco(output_dir: str, busco_results: pd.DataFrame) -> None:
         os.path.join(output_dir, "busco_results.csv"),
         index=False
     )
+    # Outputs different df for sample and feature data
     busco_results = _parse_df_columns(busco_results)
-    dfs = _partition_dataframe(busco_results, max_rows=100)
+    max_rows = 100
+
+    # Partition data frames
+    if len(busco_results["sample_id"].unique()) >= 2:
+        dfs = _partition_dataframe_sample_data(busco_results, max_rows)
+        counter_col = "sample_id"
+        assets_subdir = "sample_data"
+        tab_title = ["Sample details", "Feature details"]
+        is_sample_data = True
+
+        # Draw selectable histograms (only for sample data mags)
+        tabbed_context = {
+            "vega_summary_selectable_json":
+            json.dumps(_draw_selectable_summary_histograms(busco_results))
+        }
+    else:
+        dfs = _partition_dataframe_feature_data(busco_results, max_rows)
+        counter_col = "mag_id"
+        tab_title = ["BUSCO plots", "BUSCO table"]
+        assets_subdir = "feature_data"
+        is_sample_data = True
```

Comment:
This is confusing - why is `is_sample_data` set to True here as well?

Reply:
😅 it's a typo jaja. it should be false. I change it right away.